Source/Core/DolphinWX/CMakeLists.txt

```diff
@@ -212,6 +212,10 @@ if(wxWidgets_FOUND)
 		install(DIRECTORY ${BUNDLE_PATH} DESTINATION /Applications
 			USE_SOURCE_PERMISSIONS
 			)
+	elseif(WIN32)
+		set_target_properties(${DOLPHIN_EXE} PROPERTIES
+			WIN32_EXECUTABLE ON
```

Comment:
Ah, I see what you mean.
Still, I don't want to put dirty properties like that when it's not required. Who knows what would happen when we port to Windows Phone?